src/webots/gui/main.cpp

```diff
@@ -213,5 +213,5 @@ int main(int argc, char *argv[]) {
 #endif
 #endif
 
-  return app.exec();
+  app.exec();
 }
```

Comment:
It is strange we declare the main function as int and it doesn't return an exit status.